src/Components/Buttons/RoundGradientButton.jsx

```
@@ -1,30 +1,30 @@
-    background:
-      theme.palette.mode === "dark"
-        ? "linear-gradient(90deg, #842bd2, #ff5451, #8c52ff)"
-        : "linear-gradient(90deg, #842bd2, #ff5451, #8c52ff)",
-    content: '""',
-    zIndex: -1,
-    borderRadius: 30,
-  },
-}));
-
-export default RoundGradientButton;
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)